README.md

```diff
@@ -1 +1,68 @@
-# regression-retrieval
\ No newline at end of file
+# regression-retrieval
+
+**regression-retrieval** is a tool that runs different versions of data-retrieval utils and compares theirs resource consumption.
+
+```
+Usage:
+  regression-retrieval [OPTIONS]
+
+data-retrieval utils regression tester.
+
+This tool executes several versions of data-retrieval utils and compares query times and resource usage. There should be at least two versions specified as arguments in the following way:
+
+* v0.12.1 - release name from github (https://github.com/src-d/your-data-retrieval-util/releases). The binary will be downloaded.
+* latest - latest release from github. The binary will be downloaded.
+* remote:master - any tag or branch from repository. The binary will be built automatically.
+* local:fix/some-bug - tag or branch from the repository in the current directory. The binary will be built.
+* local:HEAD - current state of the repository. Binary is built.
+* pull:266 - code from pull request #266 from your-data-retrieval-util repo. Binary is built.
+* /path/to/your-data-retrieval-util - a binary built locally.
+
+The repositories and downloaded/built binaries are cached by default in "repos" and "binaries" repositories from the current directory.
+
+
+Application Options:
+      --binaries=     Directory to store binaries (default: binaries) [$REG_BINARIES]
+      --repos=        Directory to store repositories (default: repos) [$REG_REPOS]
+      --url=          URL to the tool repo [$REG_GITURL]
+      --gitport=      Port for local git server (default: 9418) [$REG_GITPORT]
```

Comment:
Is `--gitport` needed or is a leftover from the previous tools? Same for `--repos-file` and `--complexity`.

Reply:
I've just checked that comes from `regression.Config`. We may need to update `regression-core` to have a base config and a `GitServerConfig`.

Reply:
1) leftover
2) +

Reply:
https://github.com/src-d/regression-core/pull/19

Reply:
done